quickinstall

```diff
@@ -92,11 +114,11 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
     # OSX
     newtmp
     # Run macdeployqt to avoid linking problems
-    [[ $MACDEPLOYQT == "" ]] && MACDEPLOYQT=`find ~/Qt/*/clang_64/bin ~/Applications/Qt/*/clang_64/bin /Applications/Qt/*/clang_64/bin -name macdeployqt -print -quit 2>/dev/null`
-    $MACDEPLOYQT ckb.app >/dev/null 2>$TMPFILE
+    [[ $MACDEPLOYQT == "" ]] && MACDEPLOYQT=$(find ~/Qt/*/clang_64/bin ~/Applications/Qt/*/clang_64/bin /Applications/Qt/*/clang_64/bin -name macdeployqt -print -quit 2>/dev/null)
+    $MACDEPLOYQT ckb.app >/dev/null 2>"$TMPFILE"
```

Comment:
I think the intention here is: if `$MACDEPLOYQT` is empty, set it to something, else use it's contents...